src/ui/layout/qgslayoutdesignerbase.ui

```diff
@@ -1127,7 +1127,7 @@
    </property>
   </action>
   <action name="mActionShowPage">
-   <property name="checkable">q
```

Comment:
Apparently this 'q' was removed by QtDesigner/Creator?? Is that a vim-left over? Or did it really do something?